templates/pages/login.jsx

```diff
@@ -58,7 +59,7 @@ var Login = React.createClass({
     var wrapperClass = "centerDiv";
 
     if (this.queryObj.passwordReset) {
-      wrapperClass += " largeWrapper"
+      wrapperClass = utils.getClassNameString(wrapperClass, "largeWrapper");
```

Comment:
at this point, I'm going to say "just use `classnames` instead", because writing shim code ourselves is a little silly.
```
var classnames = require('classnames');

var className = classnames({
  centerDiv: true,
  largeWrapper: !!this.queryObj.passwordReset
});
```